app/v3/controllers/RetrieveSelfEmploymentBsasController.scala

```diff
@@ -89,9 +89,17 @@ class RetrieveSelfEmploymentBsasController @Inject()(
 
   private def errorResult(errorWrapper: ErrorWrapper) =
     errorWrapper.error match {
-      case BadRequestError | NinoFormatError | CalculationIdFormatError | RuleTypeOfBusinessIncorrectError => BadRequest(Json.toJson(errorWrapper))
-      case NotFoundError                                                                                   => NotFound(Json.toJson(errorWrapper))
-      case InternalError                                                                                 => InternalServerError(Json.toJson(errorWrapper))
-      case _               => unhandledError(errorWrapper)
+      case _ if errorWrapper.containsAnyOf(
```

Comment:
If you use scalafmt formatter (ticking reformat on save in preferences on intellij) it will format this a bit better. 

The formatting for the class declaration (lines 35-46) also looks a bit strange